doc/api_reference.md

```diff
@@ -83,6 +84,19 @@ See the [Cancellation](cancellation.md) section for more details on cancellation
 Returns a sender that transforms the value of the `predecessor` by calling
 `func(value)`.
 
+### `finally(Sender source, Sender completion) -> Sender`
+
+Returns a sender that will first launch `source` and upon completion of
+`source` will launch the `completion` sender.
+
+If `completion` completes with `set_value()` (which must ) then the
```

Comment:
"which must..." ???

Reply:
I think this was supposed to be something like "which must be invoked with an empty pack of values"??